src/translators/tests/test_crate.py

```diff
@@ -29,6 +29,21 @@ def test_insert_entity(translator, entity):
 
     check_notifications_record([entity], loaded_entities)
 
+def test_insert_same_type_entities_with_different_attrs( translator, sameTypeEntitiesWithDifferentAttrs ):
```

Comment:
Ok, good, this is one of the tests I mentioned above. There's a catch though at assertion time, see my comment on ```common.py```